### PR TITLE
Improve HTTP error handling and increase timeout

### PR DESCRIPTION
**Pull request by oschwald**

### CODE REVIEW

File: lib/WebService/PayPal/PaymentsAdvanced/Role/HasUA.pm [hunk 1]
```diff
@@ -12,7 +12,7 @@ has ua => (
     isa     => InstanceOf ['LWP::UserAgent'],
     default => sub {
         my $ua = LWP::UserAgent->new;
-        $ua->timeout(5);
+        $ua->timeout(10);
```
Comment (oalders) [inline lib/WebService/PayPal/PaymentsAdvanced/Role/HasUA.pm]: Should this be `60` rather than `10`?

File: t/PaymentsAdvanced/Response/FromHTTP.t [hunk 2]
```diff
@@ -16,22 +16,54 @@ use WebService::PayPal::PaymentsAdvanced::Response::FromHTTP;
...
+    };
 
     isa_ok(
-        exception {
-            WebService::PayPal::PaymentsAdvanced::Response::FromHTTP->new(
-                http_response => $http_response,
-                request_uri   => 'http://www.paypal.com/',
-                )
-        },
+        $ex,,
```
Comment (oalders) [inline t/PaymentsAdvanced/Response/FromHTTP.t]: Why are there two commas here?

File: lib/WebService/PayPal/PaymentsAdvanced/Role/HasUA.pm (hunk 1 above)
Comment (oschwald) [inline lib/WebService/PayPal/PaymentsAdvanced/Role/HasUA.pm]: Woops. Thanks!

File: t/PaymentsAdvanced/Response/FromHTTP.t (hunk 2 above)
Comment (oschwald) [inline t/PaymentsAdvanced/Response/FromHTTP.t]: Typo, thanks.

**Pull request merged**